src/Tools/ExternalAccess/Xaml/External/ExportXamlStatelessLspServiceAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.LanguageServer;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Xaml;
+
+/// <summary>
+/// Defines an easy to use subclass for <see cref="ExportStatelessLspServiceAttribute"/> with the Roslyn languages contract name.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class), MetadataAttribute]
+internal sealed class ExportXamlStatelessLspServiceAttribute : ExportStatelessLspServiceAttribute
```

Comment:
@mgoertz-msft is this working for you for exporting a service in the Xaml repo, that doesn't inherit from a base class defined in here Roslyn?

Maybe i've missed something in this PR that I need in mine, but I'm getting MEF composition failures because `ILspService`, which this attribute exports, is not visible to Razor.